install/share/bind.named.conf.template

```diff
@@ -21,6 +21,9 @@ options {
 	bindkeys-file "$BINDKEYS_FILE";
 
 	managed-keys-directory "$MANAGED_KEYS_DIR";
+
+	/* crypto policy snippet on platforms with system-wide policy. */
+	$INCLUDE_CRYPTO_POLICY
```

Comment:
I checked the man pages of `named.conf` but did not find a description of what happens when a file path is pasted to the config. Can you point me to docs that would explain this?